packages/eslint-config-udemy-basics/README.md

```diff
@@ -1,6 +1,26 @@
-# ESLint Config Udemy Base
+Udemy ESLint Config for Jasmine
+===============================
 
-This config is mostly used by codebases inside [`udemy/website-django`](https://github.com/udemy/website-django) repo 
-other than the [`static/`](https://github.com/udemy/website-django/tree/master/static/.eslintrc.js) 
-(e.g. [`udemy/website-django/axe`](https://github.com/udemy/website-django/tree/master/axe/.eslintrc.js), 
-[`udemy/website-django/e2e`](https://github.com/udemy/website-django/tree/master/e2e/.eslintrc.js)).
+This configuration provides a basis for any ES2015 JS codebase developed for Udemy. 
+It is mainly used by [`udemy/website-django/axe`](https://github.com/udemy/website-django/tree/master/axe/.eslintrc.js) 
+and [`udemy/website-django/e2e`](https://github.com/udemy/website-django/tree/master/e2e/.eslintrc.js) codebases.
+
+# Installation
+
+Install [ESLint](https://www.github.com/eslint/eslint) locally.
+
+    $ yarn add eslint --dev
+
+Install Udemy ESLint Plugin locally.
+
+    $ yarn add eslint-config-udemy-basics --dev
+
+# Configuration
+
+Add an `extends` section to your `.eslintrc.js` and specify `udemy-basics` as an extended configuration.
+You can additionally adjust rules by overriding them at your `rules` section.
+
```

Comment:
I feel like an example is worth 1000 lines of documentation ;)

Reply:
Agreed, I am adding examples.